consts.php

```diff
@@ -529,4 +534,59 @@
         'silverstripe-versionfeed' => '3.1',
         'silverstripe-webauthn-authenticator' => '5.1',
     ],
+    '5.2' => [
```

Comment:
This block is not required right now, but I left it as placeholder that should be updated in the future.

Reply:
If it's not required right now, please remove it. There is an explicit step in the minor release process to add these as part of the release, so we don't need to add these ahead of time or be worried about forgetting to add it.